operations/CSM_product_management/Perform_NCN_Personalization.md

```diff
@@ -1,4 +1,4 @@
-# Perfrom NCN Personalization
+# Perform NCN Personalization
```

Comment:
But perfrom is a perfectly cromulent word!

Reply:
It certainly embiggens my understanding of this procedure.